contracts/scripts/e2e/utils.ts

```diff
@@ -1,76 +1,76 @@
-const l2Provider = new ethers.providers.JsonRpcProvider(
+import * as hardhat from "hardhat"
+import { l1OracleAddress, l2PortalAddress, l2StandardBridgeAddress } from "./addresses";
+
+const l2Provider = new hardhat.ethers.providers.JsonRpcProvider(
   "http://localhost:4011"
 );
 
-const l1Provider = new ethers.providers.JsonRpcProvider(
+const l1Provider = new hardhat.ethers.providers.JsonRpcProvider(
```

Comment:
Why the switch from "ethers" to "hardhat"? I see we continue to use "ethers", so I'm guessing there's a functionality reason